config/manager/manager.yaml

```diff
@@ -28,9 +28,23 @@ spec:
       - command:
         - /manager
         args:
+        - "--health-probe-bind-address=:8081"
+        - "--metrics-bind-address=127.0.0.1:8080"
         - --enable-leader-election
         image: controller:latest
         name: manager
+        livenessProbe:
```

Comment:
Let's set the probes with these values to avoid operator pod restarting due to temporary cluster/connection issues. Some values are default (scheme), but it'll be good to explicitly set so it's clear what's being used.

```yaml
                livenessProbe:
                  failureThreshold: 3
                  httpGet:
                    path: /healthz
                    port: 8081
                    scheme: HTTP
                  initialDelaySeconds: 90
                  periodSeconds: 30
                  successThreshold: 1
                  timeoutSeconds: 10
                readinessProbe:
                  failureThreshold: 3
                  httpGet:
                    path: /readyz
                    port: 8081
                    scheme: HTTP
                  initialDelaySeconds: 20
                  periodSeconds: 30
                  successThreshold: 1
                  timeoutSeconds: 10
```